src/main/java/frc/trigon/robot/subsystems/collector/Collector.java

```diff
@@ -0,0 +1,45 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Collector extends SubsystemBase {
+    private static final WPI_TalonFX
+            openingMotor = CollectorConstants.OPENER_MOTOR,
+            collectingMotor = CollectorConstants.COLLECTOR_MOTOR;
+
+    private final static Collector INSTANCE = new Collector();
+
+    private Collector() {
+
+    }
+
+    public static Collector getInstance() {
+        return INSTANCE;
+    }
+
+    private void collect() {
+        openingMotor.set(CollectorConstants.OPENING_VOLTAGE);
+        collectingMotor.set(ControlMode.PercentOutput, CollectorConstants.COLLECTING_VOLTAGE);
+    }
+
+    private void close() {
+        openingMotor.set(CollectorConstants.CLOSING_VOLTAGE);
+        collectingMotor.disable();
+    }
+
+    private void eject() {
+        collectingMotor.set(ControlMode.PercentOutput, CollectorConstants.EJECTING_VOLTAGE);
+    }
+
+    public boolean isOpen() {
+        return openingMotor.get() > 0;
+    }
+
+    private void stop() {
+        collectingMotor.disable();
+        openingMotor.disable();
```

Comment:
You don't want to disable the motors because or at least the opening motor because that disables the break, and the opener will fall down. rather set the output to 0

Reply:
Disabling is the same as calling `set(0)`, look inside it.
Stopping the system is a method we want in case of something not good happening, not so much during a match.

Reply:
![image](https://user-images.githubusercontent.com/65887925/189651757-d2200914-68a9-41af-bd5b-368f9223c8de.png)
by my understanding "disables motor controller" means that the brake is also disabled and therefore different from `.set(0)` however if this isn't going to be used in code then it doesn't matter if the break is off...

Reply:
and BTW I did look at the code before writing that. the documentation in the code has the same explanation of the mode

Reply:
and brake/coast is independent of the mode, even when the robot is disabled the motor can break.

Reply:
I don't deny that it sets the power to 0. what I dispute is this " brake/coast is independent of the mode" if that is so then what is `ControlMode.Disable`? what does "disables the motor controller" mean?

Reply:
Same as setting the output to 0

Reply:
Then tell CRT to fix their documentation because that isn't obvious